build-tools/config/webpack/webpack.prod.conf.js

```diff
@@ -62,21 +63,21 @@ const webpackConfig = merge(baseWebpackConfig, {
     chunkFilename: path.posix.join('', config.build.versionPath + '/js/[id].js'),
     publicPath: config.build.publicPath,
   },
+  optimization: {
+    concatenateModules: true,
+    minimize: true,
+    splitChunks: {
+      chunks: 'all'
+    },
+    runtimeChunk: true
+  },
   plugins: [
     new WebpackCleanupPlugin(),
     new webpack.optimize.ModuleConcatenationPlugin(),
     new webpack.DefinePlugin({
       'process.env': env,
     }),
     new webpack.NamedChunksPlugin(),
```

Comment:
This one is automatically on in `development` mode, so can be removed.

Reply:
Ah wait, this is prod, maybe we want to leave it on here :) (but can be removed from dev config)